src/readinglistbar.cpp

```diff
@@ -6,6 +6,9 @@
 
 #include <QListWidgetItem>
 #include <QFileDialog>
+#include <QStandardPaths>
+
+const QString downloadDir = QStandardPaths::writableLocation(QStandardPaths::DownloadLocation);
```

Comment:
Is the Downloads directory a good default location? Isn't Documents a better one?

Reply:
@kelson42 I checked the chromium im/export and they use Document as the default location.

Reply:
OK, then go for "Documents"

Reply:
Done